src/streamlink/plugins/tvrby.py

```diff
@@ -21,34 +26,21 @@ class TVRBy(Plugin):
 
     def __init__(self, url):
         # ensure the URL ends with a /
-        if not url.endswith("/"):
-            url += "/"
         super(TVRBy, self).__init__(url)
 
     @classmethod
     def can_handle_url(cls, url):
         return cls.url_re.match(url) is not None
 
-    @Plugin.broken()
     def _get_streams(self):
-        res = self.session.http.get(self.url)
+        res = http.get(self.url)
```

Comment:
You need to use `self.session.http` instead of `http` here, otherwise there can be problems with `http` not being set up in some cases.